### PR TITLE
A Tutorial for viewing Box storage

### LINKED ISSUE #568: How to find total Box.com storage used
### What would you like to see added?

box_storage_used

1. Click the "Account" icon at top right.
2. In drop-down menu, click "Account Settings".
3. On the "Account" tab, scroll down to "Account Details".
4. Find the text "Storage Used". Below that should be the amount of storage used for files you own.

### PR DESCRIPTION
**Pull request by iam4tune**
# Pull Request

## Overview

Created a tutorial for users to view their Box storage size.

## Related Issues

Fixes #568

### CODE REVIEW

File: docs/data_management/box_storage.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Box Storage
+
+To find information on Box and how to log in to your Box account, please find the details [here](<hhttps://www.uab.edu/it/home/tech-solutions/file-storage/box>).
```
Comment (wwarriner) [inline docs/data_management/box_storage.md:3]: Angle brackets not needed as part of a text link, and extra `h` in the URL.

**Review by wwarriner — CHANGES_REQUESTED** [review]

File: docs/data_management/enterprise_storage.md
Comment (wwarriner) [inline docs/data_management/enterprise_storage.md:1]: Change name to `storage_alternatives.md`

Don't forget to add an entry under "Data Management" in `mkdocs.yml`.

File: docs/data_management/box_storage.md
Comment (wwarriner) [inline docs/data_management/box_storage.md:1]: Remove this file from the branch.

**Review by wwarriner — CHANGES_REQUESTED** [review]
Everything looks good, with the exception of the two comments on:

- `box_storage.md`
- `enterprise_storage.md`

Thanks Fortune!

**Review by iam4tune — COMMENTED** [review]
Addressed comments for this pull request.

**Review by wwarriner — DISMISSED** [review]

**Pull request closed without merging**